api-reference/50 Common/utils/pdfExporter/exportDataGrid(options).md

```diff
@@ -4,13 +4,226 @@ module: pdf_exporter
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Exports grid data to a PDF file.
 
 ##### return: Promise<void>
-<!-- Description goes here -->
+A <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">Promise</a> that is resolved with an object that contains the coordinates of the first and last cells.
```

Comment:
`##### return: Promise<void>`

Shouldn't it be

`##### return: Promise<CellRange>`

Reply:
Developers say that it should be ##### return: Promise<void>. Also I have to remove the following part:
>that is resolved with an object that contains the coordinates of the first and last cells.